packages/office-ui-fabric-react/src/components/Persona/PersonaPresence.styles.ts

```diff
@@ -0,0 +1,287 @@
+        width: personaPresenceSize.size12,
+        borderRadius: '50%',
+        top: 'auto',
+        right: `-${personaPresenceSize.border}`,
+        bottom: `-${personaPresenceSize.border}`,
+        border: `${personaPresenceSize.border} solid ${palette.white}`,
+        textAlign: 'center',
+        boxSizing: 'content-box',
+        MsHighContrastAdjust: 'none',
+
+        selectors: {
+          [HighContrastSelector]: {
+            borderColor: 'Window',
+            color: 'Window',
+            backgroundColor: 'WindowText',
+          }
+        }
+      },
+
+      size.isSize10 && {
+        right: 'auto',
+        top: '7px',
+        left: 0,
+        border: 0,
+
+        selectors: {
+          [HighContrastSelector]: {
+            top: '9px',
+            border: '1px solid WindowText',
+          }
+        }
+      },
+
+      (size.isSize10 || size.isSize24 || size.isSize28 || size.isSize32) && {
+        height: personaPresenceSize.size8,
+        width: personaPresenceSize.size8,
+      },
+
+      size.isSize16 && {
+        height: personaPresenceSize.size6,
+        width: personaPresenceSize.size6,
+        borderWidth: '1.5px',
+      },
+
+      (size.isSize24 || size.isSize28) && {
+        selectors: {
+          ':after': {
+            display: 'none',
+          }
+        }
+      },
+
+      size.isSize72 && {
+        height: personaPresenceSize.size20,
+        width: personaPresenceSize.size20,
+      },
+
+      size.isSize100 && {
+        height: personaPresenceSize.size28,
+        width: personaPresenceSize.size28,
+      },
+
+      presence.isAvailable && {
+        backgroundColor: presenceColorAvailable,
+
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText',
+          },
+        }
+      },
+
+      presence.isAway && {
+        backgroundColor: presenceColorAway,
+
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText',
+          }
+        }
+      },
+
+      presence.isBlocked && {
+        backgroundColor: palette.white,
+
+        selectors: {
+          ':before': {
+            content: '""',
+            width: '100%',
+            height: '100%',
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            boxShadow: `0 0 0 2px ${presenceColorBusy} inset`,
+            borderRadius: '50%',
+          },
+
+          ':after': {
+            content: '""',
+            width: '100%',
+            height: '2px',
+            backgroundColor: presenceColorBusy,
+            transform: 'rotate(-45deg)',
+            position: 'absolute',
+            top: '40%',
+            left: 0,
+          },
+
+          [HighContrastSelector]: {
+            color: palette.contrastBlackDisabled,
+            backgroundColor: 'Window',
+
+            selectors: {
+              ':before': {
+                boxShadow: `0 0 0 2px ${palette.contrastBlackDisabled} inset`,
```

Comment:
@betrue-final-final do we want shadows in high contrast? (do they even render?)
i would prefer just removing (or at least ignoring) shadows, theyre too subtle for high contrast mode

Reply:
This `box-shadow` is needed to make the blocked icon look like this
![image](https://user-images.githubusercontent.com/7110658/38341179-0a0a1baa-382c-11e8-86d1-65e50fbd1e1b.png)

I'll change it to `border` and adjust the height and width to compensate.